src/blockchain/impl/validator_registry_impl.cpp

```diff
@@ -161,5 +168,7 @@ namespace lean {
               "Validator indices for node '{}' not found in registry YAML",
               current_node_id_);
     }
+
+    metrics_->val_validators_count()->set(current_validator_indices_.size());
```

Comment:
This class contains map from "node name" to "validator index" parsed from "genesis/validators.yaml".

Connected peer count should be counted in `NetworkingImpl` class